internals/cli/cmd_run.go

```diff
@@ -275,3 +283,109 @@ func convertArgs(args [][]string) (map[string][]string, error) {
 
 	return mappedArgs, nil
 }
+
+func maybeCopyPebbleDir(srcDir, destDir string) error {
```

Comment:
I think we should reverse the args to be (dest, src) so it's like other Go copy functions: `copy` builtin, `os.Copy`, and the upcoming `os.CopyFS` function.

Reply:
Can we also please add a test for this? It won't be an end-to-end test of `pebble run`, but all the interesting stuff is in here. Would need to add a new `cmd_run_test.go` and add `var MabyeCopyPebbleDir = maybeCopyPebbleDir` to `export_test.go`.

We don't need to test `copyFS` (we can assume that's tested in Go), but the basic "not set or dir is empty" functionality here, and that a copy is happening.